src/store/modules/user/actions.ts

```diff
@@ -240,7 +240,16 @@ const actions: ActionTree<UserState, RootState> = {
   async updatePartialOrderRejectionConfig ({ dispatch }, payload) {  
     let resp = {};
     try {
-      
+      if(!await UserService.isEnumExists("BOPIS_PART_ODR_REJ")) {
+        resp = await UserService.createEnumeration({
```

Comment:
Check for errors as well, if error throw resp.data.

if (hasError(resp)) {
        throw resp.data;
}